Makefile

```diff
@@ -60,7 +60,7 @@ test-all: ## run tests on every Python version with tox
 	tox
 
 coverage: ## check code coverage quickly with the default Python
-	coverage run --source pynotify ./example/manage.py test
+	coverage run --source pynotify --omit="pynotify/migrations/*" ./example/manage.py test
```

Comment:
I want to preserve 100% code coverage - in newly introduced migration, there is code that is run only when some instances of Notifications are present. Anyway, I think that migrations don't need to be subject to code coverage, since they are guaranteed to run.